components/ImageCard.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-card class="ImageCard" :href="url">
```

Comment:
```suggestion
  <v-card
    outlined
    :href="url"
    class="ImageCard"
  >
```
忘れていました...影を無くし、周りの線を付けるためにこのようにしてください...